### DISCUSSION TITLE
Magma multi-tenancy support

### DISCUSSION THREAD

**Discussion by jimhuangh** [magma/magma#7823] [Q&A] [answered]
Magma Team, 

For Magma multi-tenancy, is specific L2/L3 interface configuration needed to deliver individual tenant data streams to different tenant customers with maximum security? 

Regards,
Jim

**Comment by karthiksubraveti** [comment, marked as answer]
No. We haven't yet considered splitting tenancy at network level.

**Reply by jimhuangh** [reply]
Karthik, 

Thanks for your clarification.

Regards,
Jim

**Reply by MerouaM** [reply]
Hello everyone,

Is this feature currently available on Magma ?

Thank you.

Best reagrds,
